### ISSUE TITLE
Can't set JWT Bearer to authorization

### ISSUE THREAD

**Issue by andreccorrea** [davibaltar/swagger-autogen#47]
Hi,

There is anyway to configure JWT Bearer token in securityDefinitions and use it on UI Express?

Thank you,

André Corrêa

**Comment by davibaltar** [comment]
Hi @andreccorrea. Thank you for opening the issue. You can try this:

In your **securityDefinitions**:
```js
const doc = {
    ...
    securityDefinitions:{
        ...
        Bearer:{
            type: "apiKey",
            in: "header",
            name: "Authorization",
            description: "any description..."
        }
    }
}
```

In your **endpoint**:
```js
app.get('/your_endpoint', (req, res) => {

    /* #swagger.security = [{
        "Bearer": []
    }] */

})
```

Tell me if not work, please.

**Comment by andreccorrea** [comment]
It didn't work.
After authorizing and trying to send a request, it keeps "Loading" an shows an error in the js console (see bellow):

`TypeError: h.forEach is not a function
    t build-request.js:91
    cr build-request.js:17
    dr index.js:259
    qe actions.js:452
    je utils.js:177
    k Redux
    p wrap-actions.js:33
    r system.js:174
    Q system.js:461
    ze actions.js:499
    je utils.js:177
    k Redux
    a spec-wrap-actions.js:9
    r system.js:174
    Q system.js:461
    n execute.jsx:66
    n execute.jsx:80
    n execute.jsx:90
    React 14
    listen EventListener.js:29
    React 74
    j root-injects.jsx:69
    m index.js:192
    Nr index.js:213
    onload swagger-ui-init.js:1471
    EventHandlerNonNull* swagger-ui-init.js:2
system.js:464:16
    Q system.js:464
    ze actions.js:499
    je utils.js:177
    k Redux
    a spec-wrap-actions.js:9
    r system.js:174
    Q system.js:461
    n execute.jsx:66
    n execute.jsx:80
    n execute.jsx:90
    React 14
    listen EventListener.js:29
    React 74
    j root-injects.jsx:69
    m index.js:192
    Nr index.js:213
    onload swagger-ui-init.js:1471
    (assíncrono: EventHandlerNonNull)
    <anonymous> swagger-ui-init.js:2
`

**Comment by andreccorrea** [comment]
If I set the security in the securityDefinitions, for all endpoints, it stucks loading and show the error I posted above.
If I set the security in the endpoint, the response is "Forbidden".

**Comment by davibaltar** [comment]
@andreccorrea I cannot reproduce this problem here.

Could you submit a simple project in your repository that reproduces just this problems and send me the link? 

That way it is easier to identify the problem.

**Comment by davibaltar** [comment]
Solved in #57.
